pkgs/development/python-modules/astropy-extension-helpers/default.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "extension-helpers";
+  format = "pyproject";
+  version = "0.1";
+
+  doCheck = false;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10iqjzmya2h4sk765dlm1pbqypwlqyh8rw59a5m9i63d3klnz2mc";
+  };
+  patches = [ ./permissions.patch ];
```

really pedantic, but follows most other packages:
```suggestion
  version = "0.1";
  format = "pyproject";

  src = fetchPypi {
    inherit pname version;
    sha256 = "10iqjzmya2h4sk765dlm1pbqypwlqyh8rw59a5m9i63d3klnz2mc";
  };
  
  patches = [ ./permissions.patch ];
    
  doCheck = false;
```